training/distributed_training/pytorch/data_parallel/yolov5/yolov5.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Clarify if the user can use any names they want, or do they need to look up this information? If they can use anything, I think you can make these commented strings the default, and state that the user can modify them as desired.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>